.github/workflows/build-artifacts-code.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Use Node.js ${{ env.NODE_VERSION }}
-        uses: actions/setup-node@v3.5.1
+        uses: actions/setup-node@v3
```

Comment:
Does this mean that the latest 3.*.* version will be taken?

Reply:
Technically, no.  Practically, yes.

Reply:
Technically this is the "v3" tag.  When GitHub issues a new release, they modify their old "v3" tag to point to the latest 3.x tag.  This is standard practice across the GitHub official repos that support workflows.  But if they were not to do this, the v3 tag would remain on the same commit.

Any developer that writes a workflow script can actually change the tag to point to anything they want.  So if for security you actually wanted to pin to a certain release/commit, without the possibility that the developer could change it on you, you'd use `@sha` where `sha` is the sha code of the commit.

Reply:
Ok, thanks.